src/client/components/insurance-information/InsuranceInformationSection.jsx

```diff
@@ -78,10 +79,30 @@ class InsuranceInformationSection extends React.Component {
               <td>{insuranceGroupCode}</td>
             </tr>
           </tbody>
+          <tbody>
```

Comment:
Good catch! Medicaid was previously part of that section be we removed it to make the sections shorter.